### DISCUSSION TITLE
@xstate/test "offline" mode?

### DISCUSSION THREAD

**Discussion by karfau** [statelyai/xstate#1607] [Ideas]
Are there any plans or ideas around generating test suites "offline" (seperate from/before test execution)?

(The term "offline" seems to be common in the area of model based testing.)

This might also be helpful to see what the actually happens when using @xstate/test, so ideally the same code can be used for generating the cases on the fly vs offline.

Or maybe there are good reasons for not providing that option?

**Comment by davidkpiano** [comment]
I think that's a really good idea, actually. It would basically be like snapshot testing for MBT.

I'm thinking about how `@xstate/test` can be improved for a version 1.0, so I'll start to make a list and add that to it.
